### ISSUE TITLE
closure-make-deps doesn't throw appropriate error when passing neither `root` nor `file`?

### ISSUE THREAD

**Issue by zaki-yama** [google/closure-library#979]
It seems that `closure-make-deps` command expects either `root` or `file` option according to here:
https://github.com/google/closure-library/blob/master/closure-deps/bin/closuremakedeps.js#L210-L214

but when passing neither, other error occurs.

```
$ npm install google-closure-deps
$ ./node_modules/.bin/closure-make-deps
Error: Could not find path to Closure. Closure's base.js either needs to be included or --closure-path provided.
    at getClosureDep (/Users/.../node_modules/google-closure-deps/bin/closuremakedeps.js:178:11)
    at main (/Users/.../google-closure-deps/bin/closuremakedeps.js:252:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:757:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

**Comment by jplaisted** [comment]
Closure's base.js file must either be under a `root`, specified as a `file`, or you can avoid passing in the actual file if you give it the `closure-path` argument.

I can attempt to make the error message clearer. Would the above text be clearer?

tl;dr in some way the tool needs to know about where Closure Library is located.

**Comment by zaki-yama** [comment]
@jplaisted 
Thank you very much for replying. Then I might have misunderstood.
I was assuming that either `root` or `file` is required even if `closure-path` is specified.
Am I wrong?

**Comment by jplaisted** [comment]
Yes you need `root` or `file`. If you only were to run with `closure-path`, and that check didn't exist, you'd produce an empty file every time!

**Comment by zaki-yama** [comment]
@jplaisted 
Sorry, I'm still confused. Are these expected behaviors?

**1) With no arguments, an error occured.**

```
$ ./node_modules/.bin/closure-make-deps
Error: Could not find path to Closure. Closure's base.js either needs to be included or --closure-path provided.
    at getClosureDep (/Users/.../node_modules/google-closure-deps/bin/closuremakedeps.js:178:11)
    at main (/Users/.../google-closure-deps/bin/closuremakedeps.js:252:22)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:757:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

But this error is not the one written here:
https://github.com/google/closure-library/blob/master/closure-deps/bin/closuremakedeps.js#L211-L212

**2) With `closure-path` argument only, get no output.**

```
$ ./node_modules/.bin/closure-make-deps --closure-path=node_modules/google-closure-library/closure/goog


```

### My thought
In both cases, since I don't pass either `root` or `file`, this error occurs.
https://github.com/google/closure-library/blob/master/closure-deps/bin/closuremakedeps.js#L211-L212

**Comment by shicks** [comment]
This issue is being closed as a preparation step before archiving the repository.  See the [README](https://github.com/google/closure-library) for more detail.
